Base/res/html/misc/private-element-test.html

```diff
@@ -0,0 +1,59 @@
+<html>
```

Comment:
Please run prettier on this file :)

Reply:
Good one, should we add html to lint-prettier.sh perhaps? Or should I really not have put the js in the html file?

Reply:
> should we add html to lint-prettier.sh perhaps?

Probably a good idea if others agree with enforcing prettier formatting for HTML also :)

> Or should I really not have put the js in the html file?

That's fine, I prefer the test cases as a single HTML file containing both the CSS and JS needed. Prettier can format all three in one, luckily!